src/Components/FilterSection/FilterSection.tsx

```diff
@@ -87,13 +98,19 @@ const FilterSection = ({
     setOtherPopoverAnchor(null);
   };
 
+  const isSomeCitizenshipFilterChecked = () => {
+    return Object.values(citizenshipFilterIsChecked).some((citizenshipFilterIsChecked) => {
+      return citizenshipFilterIsChecked === true;
+    });
+  };
```

Comment:
Can we make this not apply if only citizen is selected.
```suggestion
  const isSomeCitizenshipFilterChecked = () => {
    for (const [key, value] of Object.entries(citzenshipFilterIsChecked)) {
      if (value && key !== 'citizen') {
        return true
      }
  };
```

Reply:
We've agreed to have this highlighted by default since all of the programs that are rendered initially are for `citizens`.